content/chapter/05.tex

```diff
@@ -1,3 +1,19 @@
 %!TEX root = ../../main.tex
 
-\chapter{Ausblick}
\ No newline at end of file
+\chapter{Ausblick}
+Aufgrund der technischen Mängel und Probleme der Sensoren, welche in dieser Studienarbeit verwendet wurden, war es nicht möglich, das erhoffte Ergebnis der Arbeit zu erzielen.
```

Comment:
In diesem Satz passt das, ja. In diesem Teil der Arbeit ist das Experiment abgeschlossen, weshalb die Sensoren verwendet **wurden**.

Ich schaue noch einmal über das Kapitel drüber um eventuelle Zeitformen zu korrigieren.